app/src/main/res/values/dimens.xml

```diff
@@ -2,4 +2,9 @@
     <!-- Default screen margins, per the Android Design guidelines. -->
     <dimen name="activity_horizontal_margin">16dp</dimen>
     <dimen name="activity_vertical_margin">16dp</dimen>
+
+
+    <dimen name="button_size">40dp</dimen>
+    <dimen name="button_margin">40dp</dimen>
```

Comment:
These could be more specific, rather than `button_size` it could be `btn_choose_size`